<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUserPage</class>
 <widget class="QDialog" name="NewUserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="UserLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="RefreshBtn">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddChatBtn">
        <property name="text">
         <string>Add Chat</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>82</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="AddAccountBtn">
      <property name="text">
       <string>Add account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LogoutBtn">
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ChatsLayout"/>
  </widget>
  <widget class="QWidget" name="sendMessageW" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QTextEdit" name="messageContentInput">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMsgButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgett" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QLabel" name="chatNameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="AllMessagesW">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>369</height>
     </rect>
    </property>
    <widget class="QWidget" name="AllMessagesW1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
